blog/admin.py

```diff
@@ -0,0 +1,40 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+from django.contrib.auth.models import Group
+from blog.models import User, Post, Commentary
```

Comment:
Imports should be grouped and separated by a blank line according to standard library, external libraries, and custom imports. This practice enhances both code readability and maintainability. 

Read more about topic [here](https://peps.python.org/pep-0008/#imports)